performance_tests/15zonePVAV.idf

```diff
@@ -7872,7 +7872,7 @@
     PACKAGED VAV WITH REHEAT Supply Equipment Outlet Node;  !- Setpoint Node or NodeList Name
 
   SetpointManager:MixedAir,
-    PACKAGED VAV WITH REHEAT_HeatC SAT Manager,  !- Name
+    PACKAGED VAV WITH REHEAT_HeatC SAT Manager MA,  !- Name
```

Comment:
Duplicate SetPointManager names are no longer allowed.

Reply:
I changed the labels on this PR to include `IDDChange`, because this is changing input requirements even though the idd itself isn't changing. Will need transition for this.